ztools/iscsi/log.go

```diff
@@ -0,0 +1,14 @@
+package iscsi
+
+type Target struct {
+	Name string `json:"target_name"`
```

Comment:
Is there a reason these strings aren't omitempty? Does empty string have any meaning here?